frame/staking/src/pallet/impls.rs

```diff
@@ -92,6 +92,11 @@ impl<T: Config> Pallet<T> {
 		Self::slashable_balance_of_vote_weight(who, issuance)
 	}
 
+	/// Returns the number of filled chunks for an account.
+	pub fn chunk_slots_filled(who: &T::AccountId) -> Result<usize, Error<T>> {
+		Ok(Self::ledger(&who).ok_or(Error::<T>::NotController)?.unlocking.len())
```

Comment:
```suggestion
		Ok(Self::ledger(&who).map(|l| l.unlokcing.len()).ok_or(Error::<T>::NotController)
```

This is more rust-idiomatic. But all in all, I am not the biggest fan of this one liner function 😆

Reply:
I agree, I had more logic here before. I will remove the function